runner/tests/operator/cmo_ch/v2_1_0/qc/test_qc_operator.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from django.test import TestCase
+
+from beagle import settings
+from beagle.settings import ROOT_DIR
+from beagle_etl.models import Operator
+from file_system.models import File, FileMetadata
+from runner.operator.operator_factory import OperatorFactory
+
+
+FIXTURES = [
+    "runner/tests/operator/cmo_ch/v2_1_0/qc/files_fixture.json",
+    "runner/tests/operator/cmo_ch/v2_1_0/qc/files_metadata_fixture.json",
+    "runner/tests/operator/cmo_ch/v2_1_0/qc/ports_fixture.json",
+]
+
+COMMON_FIXTURES = [
+    "runner/fixtures/runner.pipeline.json",
+    "runner/fixtures/runner.run.json",
+    "runner/fixtures/runner.operator_run.json",
+    "file_system/fixtures/file_system.filegroup.json",
+    "file_system/fixtures/file_system.filetype.json",
+    "file_system/fixtures/file_system.storage.json",
+    "beagle_etl/fixtures/beagle_etl.operator.json",
+]
+
+
+class TestAccessQCOperator(TestCase):
```

Comment:
Looks good so far! Can we just rename this test class, since it's already in use by another test.

Reply:
Resolved here: https://github.com/mskcc/beagle/blob/feature/nucleo_qc_op/runner/tests/operator/cmo_ch/v2_1_0/qc/test_qc_operator.py#L30 . Let me know how things look!